be/src/column/chunk.cpp

```diff
@@ -197,6 +197,15 @@ void Chunk::append_selective(const Chunk& src, const uint32_t* indexes, uint32_t
     }
 }
 
+void Chunk::rolling_append_selective(Chunk& src, const uint32_t* indexes, uint32_t from, uint32_t size) {
+    DCHECK_EQ(_columns.size(), src.columns().size());
+    for (size_t i = 0; i < _columns.size(); ++i) {
+        _columns[i]->reserve(size);
```

Comment:
Is this operation useful? AFAIK this size stands for how many source items should be copied.
But destination size may be greater than this size, so I think this operation can be eliminated.

Reply:
Yes, it can be removed. IColumn::append_selective() has already reserve with right size

Reply:
Fixed